### PR TITLE
FRB: Add subsequent field

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
# Description
As FRBs may have reoccurent observation from the same source but in late times.
Following CHIME, adding "subsequent" in our alert_type core schema.

I have a question: is "initial" ok or "detection" word is better instead?

Resolves #https://github.com/nasa-gcn/gcn-schema/issues/198

### CODE REVIEW

**Review by lpsinger — COMMENTED** [review]
How is "subsequent" different from "update"?

**Comment by Vidushi-GitHub** [comment]
> How is "subsequent" different from "update"?

Update can be for "initial" one as well, if first detection has updated results.

File: gcn/notices/core/Alert.schema.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
       "description": "Indication of whether this alert refers to a recent observation (current), re-analysis of archival data (archival), a planned observation in the future (planned), a signal injection (injection), commanded trigger (commanded) or a test trigger (test)."
     },
     "alert_type": {
-      "enum": ["initial", "update", "retraction"],
-      "description": "Indication of alert sequence if multiple of same type sent"
+      "enum": ["initial", "subsequent", "update", "retraction"],
+      "description": "Indication of alert sequence if multiple of the same type are sent. Alert sequence: initial refers to the detection, followed by subsequent alerts for recurrent observations from the same source. Updates and retractions come from further analysis."
```
Comment (jracusin) [inline gcn/notices/core/Alert.schema.json:25]:
```suggestion
      "description": "Indication of alert sequence if multiple of the same type are sent. Alert sequence: initial refers to the detection, followed by subsequent alerts for recurrent detections of the same source. Updates and retractions come from further analysis."
```

**Comment by Vidushi-GitHub** [comment]
@lpsinger, @jracusin and I were agreed to add it. If this ok with you, I would merge it.

**Review by jracusin — APPROVED** [review]

**Pull request merged**